snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedExportApiTest.java

```diff
@@ -1230,6 +1232,55 @@ public void exportUnpublishedOWLExpressionRefsetMembers() throws Exception {
 		assertArchiveContainsLines(exportArchive, fileToLinesMap);
 	}
 	
+	@Test
+	public void exportRf2WithConfiguration() throws Exception {
+		
+		final Map<String, Object> exportSettings = ImmutableMap.<String, Object>builder()
+				.put("maintainerType",  Rf2MaintainerType.NRC)
+				.put("nrcCountryCode", "GB")
+				.put("extensionNamespaceId", "370137002")
+				.put("refSetLayout", Rf2RefSetExportLayout.COMBINED)
+				.build();
+		
+		String codeSystemId = "CodeSystem-WithOutSettings";
+		createCodeSystem(null, codeSystemId, exportSettings).statusCode(201);
+		
+		CodeSystem codeSystem = CodeSystemRestRequests.getCodeSystem(codeSystemId).extract().as(CodeSystem.class);
+		
+		final File exportArchive = doExport(branchPath, codeSystem.getSettings());
+		
+		final Map<String, Boolean> files = ImmutableMap.<String, Boolean>builder()
+				.put("der2_sssssssRefset_MRCMDomainSnapshot_GB", true)
+				.put("sct2_Relationship_Snapshot_GB", true)
+				.put("sct2_Description_Snapshot-en_GB", true)
+				.put("der2_sRefset_SimpleMapSnapshot_GB", true)
+				.build();
+			
+		assertArchiveContainsFiles(exportArchive, files);
+		
+	}
+	
+	@Test
+	public void exportRf2WithoutConfiguration() throws Exception {
+		
+		String codeSystemId = "CodeSystem-WithSettings";
```

Comment:
And here: `Incorrect CodeSystemId, it should be SNOMEDCT-default-rf2-export-config.`